### PR TITLE
Fix #4370: remove srw video links

### LINKED ISSUE #4370: Removing Vimeo videos from SRW app
Remove Vimeo instructional video from the SRW app. 
vimeo_video

### PR DESCRIPTION
**Pull request by gurhar1133**
Now you can turn displaying the SRW instructional video links on and off as a config option (default is on)

### CODE REVIEW

File: sirepo/feature_config.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def b(msg, dev=False):
         srw=dict(
             app_url=('/en/xray-beamlines.html', str, 'URL for SRW link'),
             mask_in_toolbar=b('Show the mask element in toolbar'),
+            show_video_links=(True, bool, 'Display instruction video links'),
```
Comment (robnagler) [inline sirepo/feature_config.py:129]: make this false by default.

**Review by robnagler — CHANGES_REQUESTED** [review]

File: sirepo/feature_config.py (hunk 1 above)
Comment (moellep) [reply sirepo/feature_config.py:129]: I thought we would keep this enabled by default when BNL hosts sirepo, and only turn it off for sirepo.com

Comment (gurhar1133) [reply sirepo/feature_config.py:129]: What's the verdict?

Comment (robnagler) [reply sirepo/feature_config.py:129]: Off. I admin BNL, and they will need updated config when their servers are turned on again. I'll make sure it's there.

David was pretty clear he didn't want it on.

**Comment by gurhar1133** [comment]
Made changes

**Comment by gurhar1133** [comment]
Any more thoughts @robnagler and @moellep ?

**Comment by moellep** [comment]
Looks good.

**Review by robnagler — APPROVED** [review]

**Pull request merged**